samcli/commands/docs/command.py

```diff
@@ -0,0 +1,51 @@
+"""
+CLI command for "docs" command
+"""
+import click
+
+from samcli.cli.main import common_options, print_cmdline_args
+from samcli.commands._utils.command_exception_handler import command_exception_handler
+from samcli.commands._utils.options import common_configuration_options
+from samcli.lib.telemetry.metric import track_command
+from samcli.lib.utils.version_checker import check_newer_version
+
+HELP_TEXT = """Launch the AWS SAM CLI documentation in a browser! This command will
+    open a page containing information about setting up credentials, the
+    AWS SAM CLI lifecycle and other useful details. From there, navigate the
+    command reference as required.
+"""
+
+
+@click.command("docs", help=HELP_TEXT)
+@common_configuration_options
+@common_options
+@track_command
+@check_newer_version
+@print_cmdline_args
+@command_exception_handler
+def cli(config_file: str, config_env: str):
+    """
+    `sam docs` command entry point
+    """
+
+    # All logic must be implemented in the ``do_cli`` method. This helps with easy unit testing
+    do_cli(
+        config_file=config_file,
```

Comment:
Thought: Actually now that I think more about it, do config-file and config-env even matter for this command? may be it might in the future if we start making docs become context-aware.